server/@types/bapv.d.ts

```diff
@@ -143,3 +143,23 @@ export type Contact = {
   addresses: Address[]
   commentText: string
 }
+
+// Visit slots, for representing data derived from VisitSessions
```

Comment:
Happy to discuss naming of these - 'slots' vs 'sessions', etc! (Might try and order this file better too once this PRs in...)

Reply:
Can go with whatever we use, one for the team 🙂